src/Magnum/OvrIntegration/Context.h

```diff
@@ -53,6 +53,17 @@ struct Error {
 };
 
 /**
+ * @brief Result of @ref Context::detect(int).
+ */
+enum class OvrDetectResultFlag: Byte {
+    ServiceRunning = 1,
+    HmdConnected = 2
+};
+
+typedef Corrade::Containers::EnumSet<OvrDetectResultFlag> OvrDetectResult;
+CORRADE_ENUMSET_OPERATORS(OvrDetectResult)
```

Comment:
Well it's the result of the `ovr_Detect` method, which makes this the result of Detect. See `static OvrDetectResult Context::detect(int)`